src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,46 @@
+<%@ page import="java.util.List" %>
```

Comment:
Also don't know if you need all the imports. When I made the jsp file didn't end up needing them even though I was accessing the "user" attribute?